Formula/thors-serializer.rb

```diff
@@ -0,0 +1,60 @@
+class ThorsSerializer < Formula
+  desc "Declarative serialization library (Json/Yaml) for C++"
```

Comment:
JSON and YAML are all caps

Reply:
Will do.

But a personal comment: It looks uglier.

Reply:
Maybe one day they will be spelled Jason and Yamel.

Reply:
💯 😈

Reply:
> spelled

Americans, sigh.